process/transactionProcessor.go

```diff
@@ -512,14 +512,8 @@ func (tp *TransactionProcessor) computeTransactionStatus(tx *transaction.ApiTran
 		}
 	}
 
-	if checkIfCompleted(allLogs) {
-		return &data.ProcessStatusResponse{
-			Status: string(transaction.TxStatusSuccess),
-		}
-	}
-
 	return &data.ProcessStatusResponse{
-		Status: string(transaction.TxStatusPending),
+		Status: string(transaction.TxStatusSuccess),
```

Comment:
this is wrong. How we are going to return the pending status?

Reply:
reverted the changes